sklearn/utils/estimator_checks.py

```diff
@@ -335,10 +337,26 @@ def _construct_instance(Estimator):
                 estimator = Estimator(Ridge())
             else:
                 estimator = Estimator(LinearDiscriminantAnalysis())
+        elif any(
+            [
+                req_param.startswith("param_")
+                for req_param in required_parameters
+            ]
+        ):
+            # dealing with SearchCV objects (i.e. GridSearchCV and
+            # RandomizedSearchCV)
+            estimator = (
+                Estimator(Ridge(), {"alpha": [0.1, 1]}),
+                Estimator(LogisticRegression(), {"C": [0.1, 1]})
```

Comment:
Returning a tuple does add more complexity to the interface of `_construct_instance`. Although, I can not currently think of a better way of doing this.

If we are going with this change, I would prefer to have a docstring for `_construct_instance`.

Reply:
I added a docstring for the moment